app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
```

Comment:
- [ ] A comment to a post cannot be created successfully as seen below. Let's ensure to improve upon this aspect.

[comment_issue.webm](https://github.com/David-Lanzz/lanzz-blog/assets/12745474/3304a2e8-fded-4944-b039-0391cc10fce7)